### PR TITLE
Edit readme - Add dependencies needed to compile a release

### PR DESCRIPTION
**Pull request by lucasMesquitaBorges**
Add dependency on readme needed to compile ppcg. Without this dependency is not possible to extract all files needed when running make.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -11,10 +11,12 @@ Requirements:
 	Unless you have some other reasons for wanting to use the svn version,
 	it is best to install the latest release (3.9).
 	For more details, see pet/README.
+- zlib1g-dev Compression library
```
Comment (Meinersbur) [inline README]: This is the name of the Ubuntu package, not the name of the project clang depends on.

**Comment by Meinersbur** [comment]
Hi,

thank you for your contribution. Please note that the upstream project is http://ppcg.gforge.inria.fr/ and patches are to be submitted to isl-development@googlegroups.com. See https://github.com/Meinersbur/isl/blob/master/doc/SubmittingPatches. If you want, I can do this for you.

I think that zlib is an indirect dpendency through clang (https://llvm.org/docs/GettingStarted.html#software)

File: README (hunk 1 above)
Comment (lucasMesquitaBorges) [reply README]: Ok, i'll fix the package name, i hadn't paid attention to this detail.

**Comment by lucasMesquitaBorges** [comment]
I fixed the zlib package description.
As I'm not too much experient with C and compiling dependencies, it took me a while to find where was my compiling problem, that's why i tought would be nice to add it on readme.
If you think it's not relevant to add it in the readme because it is a indirect clang dependency, no problem just advise me and i'll close the pull request.
But, if you see no problem to add it, i didn't understood how to submit a patch by email, so i would like you do that for me.

Thank You!

**Pull request closed without merging**